src/main/java/pl/ateam/disasteralerts/user/dto/UserDTO.java

```diff
@@ -1,11 +1,26 @@
 package pl.ateam.disasteralerts.user.dto;
 
+import jakarta.validation.constraints.*;
+
 import pl.ateam.disasteralerts.user.Role;
 
-public record UserDTO(String username,
-                      String email,
-                      String password,
-                      String phoneNumber,
-                      String location,
-                      String role) {
+public record UserDTO(
+                        @Size(min=5, max=15)
+                        @NotBlank
+                        String username,
+                        @Email @NotBlank
+                        @Max(255)
+                        String email,
+                        @NotBlank @Size(min = 6, max = 255)
```

Comment:
```suggestion
                        @NotBlank @Size(min = 6, max = 25)
```
powinno być 25 lub
```suggestion
                        @NotBlank
```
usunąć całkiem bo wynika to z regexa